src/api/v4/client.js

```diff
@@ -1,35 +1,34 @@
 var Events = require('./events');
 var Engine = require('../../engine');
 var LeafletCartoLayerGroupView = require('../../geo/leaflet/leaflet-cartodb-layer-group-view');
+var VERSION = require('../../../package.json').version;
```

Comment:
This will include the whole `package.json` inside the bundle for a single `number`.

We can define `VERSION` using [webpack define plugin](https://webpack.js.org/plugins/define-plugin/) as described [here](https://github.com/webpack/webpack/issues/237#issuecomment-40398916).

Reply:
Ok, Let's use that